### ISSUE TITLE
Security vulnerability in npm package

### ISSUE THREAD

**Issue by natke** [Microsoft/botbuilder-js#538]
## Versions
Package version: 4.0.6
Nodejs version: 9.11.1
npm version: 6.4.1
O/S: tested on Mac OS and Windows

## Describe the bug
npm audit produces:

```
npm audit
                                                                                
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Out-of-bounds Read                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ base64url                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.0.0                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ botbuilder                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ botbuilder > botframework-connector > base64url              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/658                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 moderate severity vulnerability in 409 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

## To Reproduce
1. $ npm install botbuilder
2. $ npm audit

## Expected behavior
No vulnerabilities

[bug]

**Comment by JuanAr** [comment]
Thanks @natke for taking the time to report this issue. This is already fixed in master branch since 304a626eb8ce6dc9d6c6d913c960ed8e87ff7cfa.
It will be available in the next version of the npm package soon. :)

**Comment by natke** [comment]
How soon is soon? :)
